### ISSUE TITLE
Segmentation fault when skip is defined

### ISSUE THREAD

**Issue by gkyildirim** [DaVieS007/Partial-File-Recovery#1]
Hello and thanks for this work. It seems promising so I gave a shot.

Hoping that the command is correct this gives me segmantation fault on linux aarch64:
```
./pfr64 /mnt/loopback_disks/nvmeof.disk24 /mnt/loopbackdisks/nvmeof.disk24 1 0 1 00
```

If I remove 00 it works. But I guess I need 00 (skip) since I can not write destination with this error:
```
Failed to Write Destination [1] 1!
```

**Comment by DaVieS007** [comment]
Thank you for your feedback. 

On the information that you provided I can confirm that, on linux the software fails due using uninitialized memory address when the argument is provided.

I checked and fixed the code, also rebuilt all executables.
The issue fixed on 2.1r version.

I did test as well.
![Screenshot_243](https://github.com/DaVieS007/Partial-File-Recovery/assets/12925556/6f66be90-9df7-4ace-9eac-a468e783229e)
